### PR TITLE
Disable SMT tests for ASan

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

**Comment by leonardoalt** [comment]
This is actually wrong

**Comment by leonardoalt** [comment]
Ready for review now

File: .circleci/soltest.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ get_logfile_basename() {
 }
 
 BOOST_TEST_ARGS="--color_output=no --show_progress=yes --logger=JUNIT,error,test_results/`get_logfile_basename`.xml"
-SOLTEST_ARGS="--evm-version=$EVM --evmonepath /usr/lib/libevmone.so $flags"
+SOLTEST_ARGS="${SOLTEST_ARGS} --evm-version=$EVM --evmonepath /usr/lib/libevmone.so $flags"
```
Comment (bshastry) [inline .circleci/soltest.sh]: Is the `$flags` argument for additional options to be passed to soltest? I was wondering if you could assign `--no-smt` to `$flags` and if that would work.

Comment (leonardoalt) [reply .circleci/soltest.sh]: Right, could also do that

Comment (leonardoalt) [reply .circleci/soltest.sh]: I'll try that in the other PR and then update this one accordingly

**Comment by leonardoalt** [comment]
@bshastry updated, works in the other PR as well

**Comment by bshastry** [comment]
`$flags` is a very generic name that does not make it obvious they are additional command line options to `soltest.sh`. Perhaps that's a different PR :-)

**Review by bshastry — APPROVED** [review]
LGTM if tests pass

**Pull request merged**